var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -7,7 +7,7 @@
 from spack.package import *
 
 
-class QuantumEspresso(CMakePackage):
+class QuantumEspresso(CMakePackage, CudaPackage):
```

Comment:
I don't see this necessary. QE doesn't use CUDA toolkit but relies on NVHPC.

Reply:
CudaPackage has been removed in the last commit b846a23f487af9289e4274e530a7b4715edbbe05